keyboards/ikeygai/albatross/albt_s/info.json

```diff
@@ -0,0 +1,155 @@
+{
+    "keyboard_name": "ALBT-S",
+    "manufacturer": "Ikeygai Designs",
+    "usb": {
+        "pid": "0x4154",
+        "vid": "0x494B",
+        "device_version": "0.0.2"
+    },
+    "url": "", 
+    "maintainer": "Gondolindrim",
+    "matrix_pins": {
+        "cols": ["B0", "C5", "C4", "A7", "A6", "A5", "C8", "C7", "C6", "B15", "B14", "B13", "A4", "A8", "A10", "C1", "C0"],
+        "rows": ["C11", "C9", "B12", "B10", "C3", "C2", "C10", "A15"]
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey":  true,
+        "extrakey":  true,
+        "console":   false,
+        "backlight": true,
+        "nkro":      true
+    },
+    "build": {
+        "lto": true
+    },
+    "eeprom": {
+        "driver": "wear_leveling",
+        "wear_leveling": {
+            "driver": "embedded_flash",
+            "backing_size": 4096
+        }
+    },
+    "diode_direction": "COL2ROW",
+    "backlight": {
+        "pin": "B1",
+        "levels": 20,
+        "default": {
+            "brightness": 31
+        }
```

Comment:
You cant logically have a default value larger than the number of levels. (It would also default to max brightness so also redundant)

Reply:
Something is wrong with the JSON parser in that the BRIGHTNESS value is set to 20 out of 255, so the LED defaults to a very dim brightness.

Reply:
There is no special parsing other than the mapping in `info_config.h` - the define is pretty much directly generated with the given value. I'm thinking it is more to do with the inverted logic as I noted above.

Reply:
I honestly don't know why. But for instance if I delete the default brightness definition, the brightness starts at 20, but 20 of 255